certbot/src/certbot/_internal/cert_manager.py

```diff
@@ -281,7 +284,7 @@ def human_readable_cert_info(config: configuration.NamespaceConfig, cert: storag
     certinfo.append(f"  Certificate Name: {cert.lineagename}\n"
                     f"    Serial Number: {serial}\n"
                     f"    Key Type: {cert.private_key_type}\n"
-                    f'    Domains: {" ".join(cert.names())}\n'
+                    f'    Domains: {" ".join(map(str, cert.sans()))}\n'
```

Comment:
Is "SANs" too jargony here?

Reply:
I went with "Identifiers:". No strong feelings on it! Could use something else.